app/events_blueprint/models.py

```diff
@@ -1,5 +1,42 @@
 import random
-class Events(object):
+from app import db
+
+class Events(db.Model):
+
+    """This class represents the events table."""
+
+    __tablename__ = 'events'
+
+    eventid = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255))
+    userid = db.Column(db.Integer)
```

Comment:
you might want to consider a foreign key relation to the user class instead of manually entering the id